### ISSUE TITLE
RecycleApp: Invalid response from server for accessToken

### ISSUE THREAD

**Issue by sverleysen** [pippyn/Home-Assistant-Sensor-Afvalbeheer#209]
Since I updated my HA instance to 2021.3.0 the afvalbeheer integration isn't working anymore for the RecycleApp.
I get the follow error message in my logs: 
```
Logger: custom_components.afvalbeheer.sensor
Source: custom_components/afvalbeheer/sensor.py:1081
Integration: afvalbeheer (documentation)
First occurred: 9:26:54 AM (1 occurrences)
Last logged: 9:26:54 AM

Invalid response from server for accessToken
```

My config:
```yaml
platform: afvalbeheer
wastecollector: RecycleApp
resources:
  - groente-, fruit-, en tuinafval
  - papier
  - pmd
  - restafval
postcode: 2040
streetname: street
streetnumber: 1
upcomingsensor: 1
dateformat: '%d-%m-%Y'
dateonly: 1
dateobject: 0
printwastetypes: 1
# name: ''
nameprefix: 0
builtinicons: 0
disableicons: 0
dutch: 1
```

**Comment by pippyn** [comment]
This should be fixed in v4.7.16

**Comment by sverleysen** [comment]
Sorry, didn't noticed the new version. Updated and it's working now.
